upgrade/upgrade-scripts/verify-jar

```diff
@@ -19,7 +19,9 @@
 . "${BASH_SOURCE%/*}/common.sh"
 
 IMAGE_VERSION=$(get_image_version)
+IMAGE_PATH=$(get_image_path)
```

Comment:
We should verify `IMAGE_PATH` is not empty, similar to how we verify `IMAGE_VERSION` is not empty below.